Formula/corectl.rb

```diff
@@ -13,27 +22,49 @@ class Corectl < Formula
 
   depends_on "go" => :build
   depends_on "godep" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
   depends_on :macos => :yosemite
 
   def install
     ENV["GOPATH"] = buildpath
 
+    opamroot = buildpath/"opamroot"
+    opamroot.mkpath
+    ENV["OPAMROOT"] = opamroot
+    ENV["OPAMYES"] = "1"
+
     path = buildpath/"src/github.com/TheNewNormal/#{name}"
     path.install Dir["*"]
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
     cd path do
-      system "make", "corectl", *args
-      system "make", "documentation/man"
-      bin.install "corectl"
+      system "opam", "init", "--no-setup"
+      system "opam", "pin", "add", "qcow-format",
+        "https://github.com/mirage/ocaml-qcow.git#master"
```

Comment:
@ilovezfs i can _still_ get that part - what i also _get_ is that if/when it happens that `master` and `stable` are at same step (sometimes happens) the resulting builds will be different. and they shouldn't. Also, in this (very) specific case the issue is somewhat moot - because what is actually happening is that the docker/mirage guys are doing the pinning themselves on their forked repo. 

Of course it takes me 2m to do what you want - but - again - is it that better that when the `head` and `stable` are the same two different users (one that picks `stable` and another `head`) get potentially different binaries ?